src/main/groovy/org/shipkit/gradle/notes/GithubContributorsTask.java

```diff
@@ -19,21 +20,16 @@
  * https://developer.github.com/v3/repos/commits/
  * This way, we can also fetch the most recent contributors, necessary for correct release notes information.
  */
-public class FetchContributorsTask extends DefaultTask {
+public class GithubContributorsTask extends DefaultTask {
```

Comment:
Name feedback:
- The convention for task names that we currently have in Shipkit is that verb comes first. It's not great convention but that's how we do it currently so keeping it consistent will make the code easier to navigate :)
- We use "GitHub" instead of "Github" in all other places in code.

I suggest to use "FetchGitHubContributorsTask" name instead "GithubContributorsTask".